source/rst/additive_functionals.rst

```diff
@@ -1364,10 +1364,10 @@ These probability density functions help us understand mechanics underlying the
 Multiplicative Martingale as Likelihood Ratio Process
 --------------------------------------------------------
 
-:doc:`This lecture <likelihood_ratio_process>` studies **likelihood processes** and **likelihood ratio processes**.
+`This lecture <https://python.quantecon.org/likelihood_ratio_process.html>`_ studies **likelihood processes** and **likelihood ratio processes**.
```

Comment:
@najuzilu can you please double `_` to make these anonymous so sphinx doesn't complain about duplicate labels. Thanks. 
```rst
`This lecture <https://python.quantecon.org/likelihood_ratio_process.html>`__
```